pkg/cfn/manager/create_tasks.go

```diff
@@ -92,7 +92,7 @@ func (c *StackCollection) NewUnmanagedNodeGroupTask(nodeGroups []*api.NodeGroup,
 
 // NewManagedNodeGroupTask defines tasks required to create managed nodegroups
 func (c *StackCollection) NewManagedNodeGroupTask(nodeGroups []*api.ManagedNodeGroup, forceAddCNIPolicy bool, vpcImporter vpc.Importer) *tasks.TaskTree {
-	taskTree := &tasks.TaskTree{Parallel: true}
+	taskTree := &tasks.TaskTree{Parallel: false}
```

Comment:
Why is this changed?

Reply:
The task cannot be done in parallel of the managed nodegroup creation since the auto-scaling group is created by AWS after the managed nodegroup has been created.

Reply:
Hm, I have to take a look at this, but this means that if you have multiple nodegroups this will significantly slow down eksctl since all of them will be created in parallel. I think the ASG creation is part of this task so that should be fine. Did you test it by any chance leaving it like that?

Reply:
Going to perform some testing.

Reply:
Without sequential tasks, it fails since the node group does not exist yet while we try to create the ASG.
```bash
2022-04-05 15:04:11 [✖]  couldn't get managed nodegroup details for nodegroup "test-m-4xlarge-worker-c-v21-1test": ResourceNotFoundException: No node group found for name: test-m-4xlarge-worker-c-v21-1test.
{
  RespMetadata: {
    StatusCode: 404,
    RequestID: "ca88f3b0-2c12-46e3-b577-aa7db73daef2"
  },
  ClusterName: "test",
  Message_: "No node group found for name: test-m-4xlarge-worker-c-v21-1test.",
  NodegroupName: "test-m-4xlarge-worker-c-v21-1test"
}
```

Reply:
Creating two nodegroup with the propagation, I see I have 4 sequential tasks.

I was expecting to have 2 parallels tasks corresponding to the 2 nodegroups, having inside 2 sequential tasks (ng creation + tags propagation). If you have any idea about how to do that, thanks to let me know.

For now, I have mitigated the slowness, setting back Parallel to `true` if no managednodegroup has at least one propagateASGTags set to true.

Reply:
Uh, I have no idea. :D the task system is pretty complex. I'll try to unravel it today.